packages/core/useColorMode/index.ts

```diff
@@ -180,10 +187,12 @@ export function useColorMode<T extends string = BasicColorMode>(
       let style: HTMLStyleElement | undefined
       if (disableTransition) {
         style = window!.document.createElement('style')
+        if (nonce) {
+          style.nonce = nonce
+        }
```

Comment:
@43081j maybe we want to go for `styleAttributes` here? should not be breaking for this. 
-------------
I wonder if we should use `useStyleTag` here

Reply:
that probably makes sense 👍 

`styleAttributes` here, `attributes` (eventually) in `useStyleTag`

and you're right this could use `useStyleTag`, but not necessarily in this PR